board/freescale/mx6sxsabresd/mx6sxsabresd.c

```diff
@@ -139,10 +148,19 @@ static int setup_fec(void)
 
 int board_eth_init(bd_t *bis)
 {
+	int ret;
+
 	imx_iomux_v3_setup_multiple_pads(fec1_pads, ARRAY_SIZE(fec1_pads));
 	setup_fec();
 
-	return cpu_eth_init(bis);
+	ret = cpu_eth_init(bis);
+	if (!(ret)) {
```

Comment:
Unnecessary parenthesis. Everywhere else just uses `if (!ret)`
```suggestion
	if (!ret) {
```